nav-app/src/app/matrix/technique-cell/contextmenu/contextmenu.component.html

```diff
@@ -9,6 +9,9 @@
             </div>
         </div>
         <div *ngIf="configService.getFeature('selecting_techniques')" class="contextMenu-section">
+            <div class="contextMenu-button" (click)="pinCell()">
+                pin/unpin this cell
```

Comment:
I'd recommend renaming this to `pin/unpin tooltip` for clarification and moving it to its own context menu section above the `select/add to selection/remove from selection` options, just to keep the groupings consistent with their action types.